### DISCUSSION TITLE
Не работает с опцией --hostlist

### DISCUSSION THREAD

**Discussion by micle7** [bol-van/zapret#780] [Q&A] [answered]
Не пойму почему происходит следующее:
Так сайты rutracker.org и тд. открываются

> /etc/init.d/zapret restart
>Starting daemon 3: /opt/zapret/nfq/nfqws --qnum=200 --user=daemon --dpi-desync-fwmark=0x40000000 --dpi-desync=fake,disorder --dpi-desync-ttl=1 --dpi-desync-autottl=5  --dpi-desync-split-pos=1

А когда добавляю --hostlist=/opt/zapret/block.txt перестают открываться

>
> cat /opt/zapret/block.txt                  
>navalny.com
>ntc.party
>4pda.ru
>rutracker.org
>
> /etc/init.d/zapret restart
>Starting daemon 3: /opt/zapret/nfq/nfqws --qnum=200 --user=daemon --dpi-desync-fwmark=0x40000000 --dpi-desync=fake,disorder --dpi-desync-ttl=1 --dpi-desync-autottl=5  --dpi-desync-split-pos=1 --hostlist=/opt/zapret/block.txt

>cat /opt/zapret/config                     
>AUTOHOSTLIST_RETRANS_THRESHOLD=3
>TPWS_ENABLE=0
>NFQWS_ENABLE=1
>NFQWS_PORTS_TCP=443
>NFQWS_PORTS_UDP=443
>NFQWS_TCP_PKT_OUT=$((6+$AUTOHOSTLIST_RETRANS_THRESHOLD))
>NFQWS_TCP_PKT_IN=3
>NFQWS_UDP_PKT_OUT=$((6+$AUTOHOSTLIST_RETRANS_THRESHOLD))
>NFQWS_UDP_PKT_IN=0
>NFQWS_OPT="--dpi-desync=fake,disorder --dpi-desync-ttl=1 --dpi-desync-autottl=5  --dpi-desync-split-pos=1 --hostlist=/opt/zapret/block.txt"
>MODE_FILTER=hostlist
>DISABLE_IPV6=1

`

**Comment by bol-van** [comment]
Если процесс nfqws не вываливается из-за какой-то ошибки, то значит нет нужных доменов в листе
Сайт это не только его главный домен

**Comment by Rumbiz** [comment]
ECH как вариант нет в хостлисте.

**Reply by bol-van** [reply, marked as answer]
Да, верно. cloudflare-ech.com надо в лист добавлять

**Reply by numbereleven-a** [reply]
> Да, верно. cloudflare-ech.com надо в лист добавлять

так у него все сайты не работают, на том же ntc.party нет ECH

**Comment by micle7** [comment]
Всем спасибо, действительно ntc.party работал, видать я не внимательно тестировал, больше на рутрекер ориентировался, да еще и изменения как-то с задержкой после ребута применяются, т.е. чтобы сайты заработали или перестали работать, должно пройти время после рестарта запрета с измененным конфигом.
